clients/unshaded/src/main/java/tachyon/client/TachyonFile.java

```diff
@@ -450,13 +450,21 @@ private TachyonByteBuffer readLocalByteBuffer(int blockIndex, long offset, long
    *
    * @param blockInfo The blockInfo of the block to read.
    * @return TachyonByteBuffer containing the block.
+   * @throws IOException if the underlying stream throws IOException during close().
    */
-  TachyonByteBuffer readRemoteByteBuffer(ClientBlockInfo blockInfo) {
-    // We call into the remote block in stream class to read a remote byte buffer
-    ByteBuffer buf =
-        RemoteBlockInStream.readRemoteByteBuffer(mTachyonFS, blockInfo, 0, blockInfo.length,
-            mTachyonConf);
-    return (buf == null) ? null : new TachyonByteBuffer(mTachyonFS, buf, blockInfo.blockId, -1);
+  TachyonByteBuffer readRemoteByteBuffer(ClientBlockInfo blockInfo) throws IOException {
```

Comment:
@calvinjia , should we deprecate this method (or actually, the one one calling this)?